contracts/linear/src/epoch_actions.rs

```diff
@@ -300,11 +312,17 @@ impl LiquidStakingContract {
                 amount: &U128(amount),
             }
             .emit();
+
+            PromiseOrValue::Value(())
```

Comment:
Ideally we should differentiate the 4 kind of results of running `epoch_stake()` and `epoch_unstake()`:

1. not executed: not amount to settle, no candidate found, not enough amount, etc.
2. (stake/unstake) execution failed: due to some reasons, this usually won't happen
3. (stake/unstake) execution succeeded, but sync balance failed: this should not happen if contract working correctly
4. (stake/unstake) execution succeeded, and sync balance is also OK 

But I think it's OK to categorize them into 3 return values:

1. `false`: not executed (compatible with previous design)
2. `true`: executed and all successful
3. `null`: executed but failed